### ISSUE TITLE
Can't deploy static resources

### ISSUE THREAD

**Issue by AndrewRayCode** [forcedotcom/cli#1137]
```
sfdx --version
sfdx-cli/7.112.1 darwin-x64 node-v14.17.4
```
and
```
 sfdx force:source:deploy --sourcepath force-app/main/default/staticresources/illustrations/Desert.svg
*** Deploying with SOAP API ***
ERROR running force:source:deploy:  Component conversion failed: Cannot read property 'contentType' of undefined
```

**Comment by AndrewRayCode** [comment]
Could be because it's in a nested folder. Even though the command takes a file path.

**Comment by WillieRuemmele** [comment]
Hi @AndrewRayCode I confirmed that it's due to the StaticResource being inside of a directory

**Comment by uip-robot-zz** [comment]
This issue has been linked to a new work item: [W-9766036](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07AH000000UHBsYAO/view)

**Comment by WillieRuemmele** [comment]
Hi @AndrewRayCode was this working for you on 7.110.0? I'm unable to deploy a static resource inside of a directory like above

```
 ➜  sfdx which force:source:deploy                                                  
=== force:source:deploy
plugin: salesforce-alm
➜  dreamhouse-lwc git:(main) ✗  hub:(DevHub) scratch:(test-05a6t5a66a7n@example.com)
 ➜  sfdx force:source:deploy -p force-app/main/default/staticresources/test/pic.jpeg
ERROR running force:source:deploy:  We're unable to complete this action due to a missing source file. Verify that the source file name that corresponds to the /var/folders/28/dmr8rt4d5f5bq_ttscbspz580000gp/T/sdx_sourceDeploy_pkg_1629218383285/staticresources/pic.resource exists and is correct, and try again.
``` 
and with `plugin-source`
```
➜  dreamhouse-lwc git:(main) ✗  hub:(DevHub) scratch:(test-05a6t5a66a7n@example.com)
 ➜  sfdx which force:source:deploy                                                  
=== force:source:deploy
plugin: @salesforce/plugin-source
➜  dreamhouse-lwc git:(main) ✗  hub:(DevHub) scratch:(test-05a6t5a66a7n@example.com)
 ➜  sfdx force:source:deploy -p force-app/main/default/staticresources/test/pic.jpeg
*** Deploying with SOAP API ***
ERROR running force:source:deploy:  Component conversion failed: Cannot read property 'contentType' of undefined
```

while the error message is certainly much less helpful, this isn't new behavior

**Comment by WillieRuemmele** [comment]
Hi @AndrewRayCode this should be fixed in the latest-rc build of the cli 1.115.0, you can get it by either sfdx update stable-rc or npm install sfdx-cli@latest-rc --global depending on your preferred install method.

Please let us know if you're still seeing this in 1.115.0 of the CLI
